pkg/machines/scripts/create_machine.sh

```diff
@@ -41,10 +26,17 @@ else
     DISK_OPTIONS="size=$STORAGE_SIZE,format=qcow2"
 fi
 
-GRAPHICS_PARAM="--graphics vnc,listen=127.0.0.1"
-if spiceSupported; then
+DOM_GRAPHICS_CAPABILITIES="$(virsh domcapabilities | awk "/<graphics supported='yes'/{flag=1; next}/<\/graphics/{flag=0}flag")"
```

Comment:
This is indeed a much more elegant way than my `qemu  --qmp` query, nice!

The awk command could do with some spaces for readability:

    /<graphics supported='yes'/ {flag=1; next};  /<\/graphics/ {flag=0}; flag"